### PR TITLE
[fix](chore) should not ignore returned Status

### PR DESCRIPTION
**Pull request by jacktengg**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

File: be/src/vec/functions/url/functions_url.h [hunk 1]
```diff
@@ -73,8 +73,8 @@ using Pos = const char*;
   */
 template <typename Extractor>
 struct ExtractSubstringImpl {
-    static void vector(const ColumnString::Chars& data, const ColumnString::Offsets& offsets,
```
Comment (yiguolei) [inline be/src/vec/functions/url/functions_url.h:76]: why change this?

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by jacktengg** [comment]
run buildall

File: be/src/pipeline/exec/set_sink_operator.cpp [hunk 2]
```diff
@@ -88,21 +88,20 @@ Status SetSinkOperatorX<is_intersect>::_process_build_block(
...
                 using HashTableCtxType = std::decay_t<decltype(arg)>;
                 if constexpr (!std::is_same_v<HashTableCtxType, std::monostate>) {
                     vectorized::HashTableBuild<HashTableCtxType, is_intersect>
                             hash_table_build_process(&local_state, rows, raw_ptrs, state);
-                    static_cast<void>(hash_table_build_process(arg, local_state._arena));
+                    return hash_table_build_process(arg, local_state._arena);
                 } else {
                     LOG(FATAL) << "FATAL: uninited hash table";
                     __builtin_unreachable();
+                    return Status::InternalError("FATAL: uninited hash table");
```
Comment (wangbo) [inline be/src/pipeline/exec/set_sink_operator.cpp:101]: Why return after unreachable

Comment (jacktengg) [reply be/src/pipeline/exec/set_sink_operator.cpp:101]: Compiler will complain if not.

**Review by wangbo — APPROVED** [review]
LGTM

**Pull request closed without merging**